Add LDAP in system requirement and Update RBAC doc structure (#971)
SUSE/doc-caasp: adoc/admin-security-rbac-user-access.adoc

kube-system   kube-proxy-62hls                     1/1     Running   4          46d
kube-system   kube-proxy-fhswj                     1/1     Running   0          46d
kube-system   kube-proxy-r4h42                     1/1     Running   1          39d
kube-system   kube-proxy-xsdf4                     1/1     Running   0          39d
kube-system   kube-scheduler-my-master             1/1     Running   13         46d
----

If the user does not have the right permissions to access a resource,
they will receive a `Forbidden` message.

----
Error from server (Forbidden): pods is forbidden
----

== OIDC Tokens

The kubeconfig file (`kubeconf.txt`) contains the OIDC tokens necessary to perform authentication and authorization in the cluster.
OIDC tokens have an *expiration date* which means that they need to be refreshed after some time.

[IMPORTANT]
====
If you use the same user in multiple `kubeconfig` files distributed among multiple machines,
this can lead to issues. Due to the nature of access and refresh tokens (https://tools.ietf.org/html/rfc6749#page-10) only one of the machines will be fully able to refresh the token set at any given time.

> The user will be able to download multiple `kubeconfig` files, but they will only work until one of them needs to refresh the session.

Reviewer: This block is a little hard to read:

`The user will be able to download multiple 'kubeconfig' files, but they will only work until one of them needs to refresh the session.
After that, only one machine will work, namely the first machine which refreshed the token.`

Suggest to replace with:
`The user will be able to download multiple 'kubeconfig' files. However, the file with the same user is likely to be valid only for single access until expiration.`

After that, only one machine will work, namely the first machine which refreshed the token.

> Dex regards one session per user and refreshes `id-token` and `refresh-token` together.

Reviewer: ```suggestion
Dex regards one session per user. The `id-token` and `refresh-token` are refreshed together.
```

If a second user is trying to login to get a new `id-token`, Dex will invalidate the previous `id-token` and `refresh-token` for the first user.

> The first user will still be able to use the old `id-token` until expiration but after that, the first user is not allowed to refresh the `id-token` with the now invalid `refresh-token`.

Reviewer: ```suggestion
The first user is still able to continue using the old `id-token` until expiration. After expiration, the first user is not allowed to refresh the `id-token` due to the invalid `refresh-token`.
```

> Only the second user will have a valid `refresh-token`. You will encounter an error like: `"msg="failed to rotate keys: keys already rotated by another server instance"`.

Reviewer: ```suggestion
Now, only the second user will have a valid `refresh-token`. The first user will encounter an error like: `"msg="failed to rotate keys: keys already rotated by another server instance"`.
```

If sharing the same `id-token` in many places, all of them can be used until expiration.

> The first user refreshing the `id-token` & `refresh token` will be able to continue accessing the cluster until the tokens expire.

Reviewer: ```suggestion
The first user to refresh the `id-token` and `refresh token` will be able to continue accessing the cluster.
```
